src/client/render/factory.cpp

```diff
@@ -23,30 +23,63 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "plain.h"
 #include "anaglyph.h"
 #include "interlaced.h"
-#include "pageflip.h"
 #include "sidebyside.h"
+#include "secondstage.h"
+#include "client/shadows/dynamicshadowsrender.h"
+
+struct CreatePipelineResult
+{
+	v2f virtual_size_scale;
+	ShadowRenderer *shadow_renderer;
+	RenderPipeline *pipeline;
+};
+
+void createPipeline(const std::string &stereo_mode, IrrlichtDevice *device, Client *client, Hud *hud, CreatePipelineResult *result);
```

Comment:
same here for `CreatePipelineResult&`
(doesn't matter that much here though)

Reply:
fixed